src/network/netdev/batadv.c

```diff
@@ -0,0 +1,199 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <inttypes.h>
+
+#include "netdev/batadv.h"
+#include "network-internal.h"
+#include "networkd-manager.h"
+#include "netlink-util.h"
+#include <linux/genetlink.h>
```

Comment:
please keep public header includes (i.e. the ones with `<>`) before privat header includes (i.e. the ones with `""`)